### PR TITLE
Update recommended node version

### PR DESCRIPTION
**Pull request by vcarl**
Thanks @jakeurban for reporting!

### CODE REVIEW

**Review by JakeUrban — APPROVED** [review]
LGTM!

**Comment by JakeUrban** [comment]
Btw, I'm using 14 and it doesn't have any issues. We may just want to mention that your node version should stay up to date.

**Review by piyalbasu — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -33,8 +33,11 @@
 
 To build this project, you must have the following dependencies installed:
 
-- `node 10.16.3`
-  - The latest node version may not be compatible with the "sharp" module
+- `node v13.12.0`
```
Comment (vcarl) [inline README.md:36]:
```suggestion
- A modern version of node. We recommend the current LTS.
```

Comment (vcarl) [reply README.md:36]: Good call @jakeurban. Changing to this

**Pull request merged**